main.css

```diff
@@ -1,419 +0,0 @@
-}
-
-/*--------------------------------------------------------------*/
-/* Main Articles Styles */
-
-#main{
-    padding: 5px;
-    margin: 10px;
-    grid-area: main;
-    height: 100vh;
-    overflow: scroll;   
-}
-.continueReading{
-    font-size: 12px;
-    margin-bottom: 15px;
-    max-width: 100px;
-    height: 20px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-color:rgb(112, 138, 153);  
-    box-shadow: 1px 1px 1px black;
-    color: white;
-}
-.continueReading:hover{
-    box-shadow: 2px 2px 2px black
-}
-.articleImgs{
-    max-width:35vw;
-    height: auto;
-    border: rgb(179, 218, 241) solid 2px;
-}
-.LearnMore{
-    font-size: 12px;
-    margin-bottom: 15px;
-    max-width: 100px;
-    height: 20px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-color: rgb(112, 138, 153); 
-    box-shadow: 1px 1px 1px black;
-    color: white;
-}
-.LearnMore:hover{
-    box-shadow: 2px 2px 2px black;
-}
-
-
-/*--------------------------------------------------------------*/
-/* Blog Post Styles */
-a{color: inherit;}
-#features{
-    padding: 5px;
-    margin: 10px;
-    grid-area: features;
-    height:100vh;
-    overflow: scroll;
-}
-.date{
-    margin-top: 15px;
-    display: flex;
-    background-color: rgb(66, 104, 124);
-    align-items: center;
-    justify-content: center;
-    max-width: 75px;
-    height: 25px;
-    box-shadow: 1px 1px 1px black;
-    border-radius: 5%;
-    color: white;
-    
-}
-.date:hover{
-    background-color: rgb(47, 75, 90);
-    box-shadow: 2px 2px 2px black;
-}
-
-
-
-/*--------------------------------------------------------------*/
-/* Footer Styles */
-
-#footer{
-    grid-area: footer;
-    height: 600px;
-    display: flex;
-    align-content: flex-end;
-    justify-content: flex-end;
-    background-repeat: no-repeat;
-    width: 100vw;
-    color: white;
-    padding: 10px;
-    display: flex;
-    align-items: flex-end;
-    justify-content: flex-end;
-}
-
-  /* Style inputs */
-input[type=text], select, textarea {
-    width: 80%;
-    padding: 12px;
-    border: 1px solid #ccc;
-}
-input[type=submit] {
-    background-color: lightblue;
-    color: white;
-    border: none;
-    cursor: pointer;
-}
-  
-input[type=submit]:hover {
-    background-color:rgb(124, 198, 223);
-}
-  
-.column {
-    border-radius: 10%;
-    display: flex;
-    justify-content: center;
-    align-content:center; ;
-    background-color:rgb(46, 136, 167);
-    background-image: url('imgs/IMG_7188.jpg');
-    background-blend-mode:multiply;
-    font-weight: bold;
-    width: auto;
-    margin-right: 10vw;
-    margin-top: 5vh;
-    padding: 20px;
-    }
-    input[type=file]{
-     color: transparent;
-
-}
-
-
-
-/*--------------------------------------------------------*/
-/* Container for blog page styles */
-
-#blogContainer{
-    display: grid;
-    grid-gap: 0;
-    grid-template-columns: 20vw 20vw 20vw 20vw 20vw;
```

Comment:
By making your page take up 100% of the view width, you are not allowing room for padding, margins, and borders which is why you've got a little side to side wiggle at all view sizes.